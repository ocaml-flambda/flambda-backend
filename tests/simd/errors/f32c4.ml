open Stdlib

type t = float32x4

external const4 : float -> float -> float -> float -> t = "" "caml_float32x4_const4"
        [@@noalloc] [@@unboxed] [@@builtin]

let _ = const4 0.0 0.0 (Sys.opaque_identity 1.0) 0.0
