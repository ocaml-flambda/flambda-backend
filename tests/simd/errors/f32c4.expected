File "f32c4.ml", line 1:
Error: Did not find constant float arguments for caml_float32x4_const4
